docs/tutorials/fundamentals/installation.md

```diff
@@ -46,12 +46,6 @@ You may also want:
 This is the easiest way to install napari if you only wish to use it as a standalone GUI app.
 This installation method does not have any prerequisites.
 
-```{important}
-Note that the bundled app is still
-in active development, and may not be very stable. We strongly recommend
-[installing as a Python package instead](#install-as-python-package-recommended).
-```
-
```

Comment:
Why is this removed? My understanding is that this is still the case for the conda bundle too, at least to some degree?